src/pudl/etl/__init__.py

```diff
@@ -113,7 +122,13 @@
     "out_state_demand_ferc714": [pudl.analysis.state_demand],
 }
 
-all_asset_modules = raw_module_groups | core_module_groups | out_module_groups
+automation_assets = {
```

Comment:
`automation` seems like a vague name for this - we could just call them `ferceqr_deployment_assets` or something until we figure out what else we're putting in here...

Reply:
Changed to `ferceqr_deployment_assets`